### PR TITLE
[Blog] Spotlight for k8s upstream training in Japan

### PR DESCRIPTION
**Pull request by Okabe-Junya**
xref. https://github.com/kubernetes/contributor-site/pull/541

### Description

We are the organizing team for k8s upstream training in Japan. see https://github.com/kubernetes-sigs/contributor-playground/tree/master/japan for more details
We would like to feature our efforts in k/website blog!

**Important Date**

- We plan to move this PR out of draft status by ~~early October~~
- We would like to publish this blog post within ~~October~~ early November (November 10?)

Let me know if you’d like any adjustments!

Deploy preview: https://deploy-preview-48039--kubernetes-io-main-staging.netlify.app/blog/2024/10/28/k8s-upstream-training-japan-spotlight/

### Issue

Closes: #

### CODE REVIEW

**Comment by shu-mutou** [comment]
@Okabe-Junya Thanks for submittin this!
LGTM for now. I look forward to hearing more about your interview! :smile:

**Comment by atoato88** [comment]
LGTM.
Thank you for creating this blog post.

**Comment by ziyi-xie** [comment]
LGTM Thank you so much for the amazing content you created!

**Comment by Okabe-Junya** [comment]
/retitle [Blog] Spotlight for k8s upstream training in Japan

**Comment by Okabe-Junya** [comment]
Special thanks @mochizuki875 @ffjlabo !!

**Review by sftim — COMMENTED** [review]
Thanks for the PR @Okabe-Junya 

I think the best place for this is https://k8s.dev/blog/; we can still publish a mirror of the article to the main Kubernetes blog.

/hold
to flag that.

OK to unhold if:
- a publication date is agreed, and is in the future
- the article has LGTM
- if the article is mirrored from the contributor blog, the upstream article has /lgtm **and** /approve

---

PS if you would like to submit a localized (Japanese) version of the mirrored article, you can do that too. We still need an English-only original for the contributor site.

**Comment by sftim** [comment]
The November 10th publication target date is plausible, but because there is a very small pool of reviewers for both sites, I don't think it's likely.

**Comment by chris-short** [comment]
I feel like Contributor Comms can help make that happen.

Cc @kaslin @fsmunoz @mfahlandt

**Comment by Okabe-Junya** [comment]
Thank you, Tim and chris-short!

I see, considering our topic, it makes sense to post it on https://k8s.dev/blog/.
I would like to create a PR later to post it on k8s.dev

> but because there is a very small pool of reviewers for both sites, I don’t think it’s likely.

Ah, I saw a post about this topic on Slack.
Of course, that’s not an issue. I want to take this opportunity to once again express my gratitude to you and your team for the tremendous contributions in running k/website blog.

**Comment by chris-short** [comment]
> OK to unhold if: ...

And the canonical url is the K8s.dev site, right @sftim?

**Comment by Okabe-Junya** [comment]
Opened https://github.com/kubernetes/contributor-site/pull/541! Thanks!

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (sandeepkanabar) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:8]:
```suggestion
  Organizing team of Kubernetes Upstream Training in Japan
```

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (sandeepkanabar) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:63]:
```suggestion
Keita Mochizuki is a contributor who consistently contributes to Kubernetes and related projects.
```

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (sandeepkanabar) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:64]: Nits: How about adding a link to the book here? 

@sftim - would that be considered as advertising and violating the guidelines?

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (sandeepkanabar) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:65]: A link to the roadmap will be very helpful.

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (sandeepkanabar) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:144]: I guess this Question and its Answer can be better combined with previous question and its answer.

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (sandeepkanabar) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:172]:
```suggestion
[Kubernetes celebrated its 10th anniversary](https://kubernetes.io/blog/2024/06/06/10-years-of-kubernetes/) this year, and for the community to become even more active, it's crucial for people across the globe to continue contributing.
```

**Review by sandeepkanabar — CHANGES_REQUESTED** [review]
Excellently crafted blog. Loved it. Have requested few changes.

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (Okabe-Junya) [reply content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:64]: Yes, I _intentionally_ avoided this when I wrote this article — to ensure it wouldn’t be perceived as advertising or promotion.

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (Okabe-Junya) [reply content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:144]: That’s a good point. 

However, I’m not a big fan of paragraphs becoming too long — especially in interviews. 
Since this isn’t a major change, I’d like to hear another opinions from other reviewers as well. 

Thank you for your comment!

**Comment by Okabe-Junya** [comment]
Thank you for your review, @sandeepkanabar !!

**Review by ArvindParekh — COMMENTED** [review]
Looks good - great read! Thanks for the effort on this!
Just one nit - shouldn't we use Sentence Case for the headings? (Ref - [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#headings))

**Comment by sftim** [comment]
> Looks good - great read! Thanks for the effort on this! Just one nit - shouldn't we use Sentence Case for the headings? (Ref - [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#headings))

- Please provide feedback on https://github.com/kubernetes/contributor-site/pull/541 (which this PR mirrors)
- but yes, [having the] headings in sentence case is better

**Comment by ArvindParekh** [comment]
Thanks, Tim! Frederico has already suggested the required edits [there](https://github.com/kubernetes/contributor-site/pull/541#pullrequestreview-2389580391) :)

**Comment by fsmunoz** [comment]
Yes, it was the only thing I could think that was an improvement 😀

File: content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md
Comment (fsmunoz) [inline content/en/blog/_posts/k8s-upstream-training-japan-spotlight/index.md:5]:
```suggestion
date: 2024-10-28
```

File: content/en/blog/_posts/2024-10-28-k8s-upstream-training-japan-spotlight/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-10-28-k8s-upstream-training-japan-spotlight/index.md:8]: I think this will make more sense
```suggestion
  [Junya Okabe](https://github.com/Okabe-Junya) (University of Tsukuba) / 
  Organizing team of Kubernetes Upstream Training in Japan
```

File: content/en/blog/_posts/2024-10-28-k8s-upstream-training-japan-spotlight/index.md
Comment (sftim) [inline content/en/blog/_posts/2024-10-28-k8s-upstream-training-japan-spotlight/index.md:5]:
```suggestion
date: 2024-10-28
canonicalUrl: https://www.k8s.dev/blog/2024/10/28/k8s-upstream-training-japan-spotlight/
```

**Comment by sftim** [comment]
Previewed locally; all looks good

/lgtm
/approve
/hold cancel

**Pull request merged**
